File: 0633-sum-of-square-numbers/0633-sum-of-square-numbers.js
/**
 * @param {number} c
 * @return {boolean}
 */
var judgeSquareSum = function(c) {
    for (let i = 0; i * i <= c; i++) {
        let a = Math.sqrt(c - i * i);
        if (a === Math.floor(a)) return true;
    }
    return false;
};

File: 0633-sum-of-square-numbers/0633-sum-of-square-numbers.cpp
class Solution {
public:
    bool judgeSquareSum(int c) {
        for(long i = 0; i*i <=c; i++) {
            double a = sqrt(c - i*i);
            if(a == (int)a) return true;
        }
        return false;
    }
};